### ISSUE TITLE
In blog posts have tags at the top

### ISSUE THREAD

**Issue by maelle** [ropensci/roweb2#226]
I wonder whether it'd be better to make tags visible at the top of each post (under the rest of the metadata) as opposed to at the bottom as is now the case.

**Comment by sckott** [comment]
yeah, worth trying to see how it looks at least, and make sure we look on mobiile too

**Comment by maelle** [comment]
I'll tackle that after my big metadata/template PR.
